### PR TITLE
Nodemon

### PR DESCRIPTION
**Pull request by rachel-ftw**

### CODE REVIEW

File: routes/items.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ router.get( '/', ( request, response ) => {
   const Item = request.app.get( 'models' ).Item
 
   const { user, query } = request
-
+  console.log('NOOOOOOOOOO')
```
Comment (jrob8577) [inline routes/items.js]: Please remove console.log

File: public/bulls-eye.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jrob8577) [inline public/bulls-eye.svg]: What is this file?  Please remove if it's not used in the current PR (it looks like it was for the bullets you were working on?).

File: package.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "start": "nf start",
+    "start:dev": "nf start -f Profile_dev",
```
Comment (jrob8577) [inline package.json]: Typo here, should be `Procfile`, not `Profile`.  :P Also, please rename file to Procfile.dev

File: Procfile [hunk 4]
```diff
@@ -1 +1 @@
-web: bin/www
+web: node bin/www
```
Comment (jrob8577) [inline Procfile]: Does this need to happen? I think it's an executable file (it's permissions are +x, and it has the she-bang `#!` at the beginning of the file), so the OS knows how to execute it without explicitly providing the interpreter.

**Comment by rachel-ftw** [comment]
I ... think... I did all your requests and we're good? 
...?

**Pull request merged**